src/backend/deploy.js

```diff
@@ -1,64 +1,67 @@
+/* eslint-disable no-empty-label */
 'use strict';
-var distHelper = require('./dist.js');
-var fs = require('fs');
-var Github = require('github');
-var gh = new Github();
-var async = require('async');
-
-function encode(file) {
-  var bitmap = fs.readFileSync(file);
-  return new Buffer(bitmap).toString('base64');
-}
-
-var Gemfile = __dirname + '/Gemfile';
 
-var walk = function(dir, done) {
-  var results = [];
+const distHelper = require('./dist.js');
+const fs = require('fs');
+const Github = require('github');
+const gh = new Github();
+const async = require('async');
+let results, bitmap, i, file, fullPath, eventName, repoName, total, counter, elem, fileName, sha;
+const Gemfile = __dirname + '/Gemfile';
+const walk = function(dir, done) {
+  results = [];
   fs.readdir(dir, function(err, list) {
-    if (err) return done(err);
-    var i = 0;
+    if (err) {
+      return done(err);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Expected indentation of 12 spaces but found 6.](https://app.codacy.com/manual/triplez/open-event-webapp/pullRequest?prid=3226522)